### ISSUE TITLE
empty-payload for false positive

### ISSUE THREAD

**Issue by junaruga** [rpminspect/rpminspect#91]
As I got following email by CI, it's about empty-payload, I report here.

```
..
empty-payload: NEEDS_INSPECTION
...
In case of obvious issues with an RPMinspect test, please report a bug:
https://github.com/dcantrell/rpminspect/issues/new
```

```
   "empty-payload": [
     {
       "result": "VERIFY",
       "waiver authorization": "Anyone",
       "message": "New package XXXXXXXXXXX.rpm is empty (no payloads)",
       "remedy": "Check to see if you eliminated a subpackage but still have the %%package and\/or the %%files section for it."
     },
   ...
   ],
```

But this RPM is about SCL's meta package `rh-rubyNN.spec`. And seeing other SCL meta packages too. The empty `%files` is common practice.

rh-rubyNN.spec

```
%files

%files runtime
%doc README LICENSE
```

**Comment by dcantrell** [comment]
This looks like something that needs to go in the 'scl' profile for rpminspect.  I am in the middle of adding in config file support to enable/disable different inspections.  For the scl profile, we can just disable the emptyrpm inspection.  Right now the only way to do that is via command line options, so I just need to finish up the patch for having it in the config file and the profile can be updated.

As an example, here's the scl profile from rpminspect-data-fedora:

https://github.com/rpminspect/rpminspect-data-fedora/blob/master/profiles/scl.conf

There's also an internal rpminspect-data-redhat with RHEL-specifics.  In the Fedora case, I would add a block to scl.conf like this:

[inspections]
emptyrpm = off

I can do that once I finish up the config file patch I started yesterday.  For future profile adjustments or creating new profiles, do feel free to send PRs to the rpminspect-data-VENDOR project.

**Comment by dcantrell** [comment]
I am making this change to the scl profile in rpminspect-data-fedora and rpminspect-data-redhat.  Those are separate projects, but just noting it here for completion.  Thanks.
